core/src/main/java/com/linecorp/armeria/server/logging/AccessLogFormats.java

```diff
@@ -0,0 +1,353 @@
+ * Pre-defined access log formats and the utility methods for {@link AccessLogComponent}.
+ */
+public final class AccessLogFormats {
+
+    private static final AccessLogComponent BLANK = ofText(" ");
+
+    /**
+     * A common log format.
+     */
+    public static final List<AccessLogComponent> COMMON =
+            ImmutableList.<AccessLogComponent>builder().add(ofCommon(REMOTE_HOST))
+                                                       .add(BLANK)
+                                                       .add(ofCommon(RFC931))
+                                                       .add(BLANK)
+                                                       .add(ofCommon(AUTHENTICATED_USER))
+                                                       .add(BLANK)
+                                                       .add(ofCommon(REQUEST_TIMESTAMP))
+                                                       .add(ofText(" \""))
+                                                       .add(ofCommon(REQUEST_LINE))
+                                                       .add(ofText("\" "))
+                                                       .add(ofCommon(RESPONSE_STATUS_CODE))
+                                                       .add(BLANK)
+                                                       .add(ofCommon(RESPONSE_LENGTH))
+                                                       .build();
+
+    /**
+     * A combined log format.
+     */
+    public static final List<AccessLogComponent> COMBINED =
+            ImmutableList.<AccessLogComponent>builder().add(ofCommon(REMOTE_HOST))
+                                                       .add(BLANK)
+                                                       .add(ofCommon(RFC931))
+                                                       .add(BLANK)
+                                                       .add(ofCommon(AUTHENTICATED_USER))
+                                                       .add(BLANK)
+                                                       .add(ofCommon(REQUEST_TIMESTAMP))
+                                                       .add(ofText(" \""))
+                                                       .add(ofCommon(REQUEST_LINE))
+                                                       .add(ofText("\" "))
+                                                       .add(ofCommon(RESPONSE_STATUS_CODE))
+                                                       .add(BLANK)
+                                                       .add(ofCommon(RESPONSE_LENGTH))
+                                                       .add(ofText(" \""))
+                                                       .add(ofRequestHeader(HttpHeaderNames.REFERER))
+                                                       .add(ofText("\" \""))
+                                                       .add(ofRequestHeader(HttpHeaderNames.USER_AGENT))
+                                                       .add(ofText("\" \""))
```

Comment:
I'm curious if the header value contains `"`. Do we need any escaping?

Reply:
Thanks. I checked `apache` access log. `"` is escaped by `\`. I should fix it. :-)